src/load-config.js

```diff
@@ -91,6 +91,23 @@ const finalConfig = {
           ...(config.https || {}),
         }
       : config.https,
+  upload:
+    process.env.ACTUAL_FILE_SYNC_LIMIT ||
+    process.env.ACTUAL_SYNC_ENCRYPTED_FILE_LIMIT ||
+    process.env.ACTUAL_FILE_LIMIT
+      ? {
+          fileSyncLimit:
+            +process.env.ACTUAL_FILE_SYNC_LIMIT ||
+            +process.env.ACTUAL_FILE_SIZE_LIMIT_MB ||
+            20,
+          syncEncryptedFileLimit:
+            +process.env.ACTUAL_SYNC_ENCRYPTED_FILE_LIMIT ||
+            +process.env.ACTUAL_FILE_SIZE_LIMIT_MB ||
+            50,
+          fileLimit: +process.env.ACTUAL_FILE_LIMIT || 20,
```

Comment:
Some of these vars are still using the old name.

Reply:
Good catch, resolved